packages/feedback-service/src/helpers.ts

```diff
@@ -0,0 +1,16 @@
+// write your helper functions here
+import { RedisPubSub } from 'graphql-redis-subscriptions';
+export const pubsub = new RedisPubSub();
+const nodemailer = require("nodemailer");
+
+export const getTransporter =  nodemailer.createTransport({
+    host: "smtp.corp.redhat.com",
+    port: 587,
+    secure: false,
+    tls: {
+      // do not fail on invalid certs
+      rejectUnauthorized: false
+    },
+    logger: true,
+    debug: false // include SMTP traffic in the logs
+  });
```

Comment:
The nodemailer is not needed in the framework. Most of the email communication will be handled by JIRA, and if we need custom emails even after that, you can use the Notifications Microservice to send out the emails.